cmd/app/serve.go

```diff
@@ -437,7 +460,7 @@ func StartDuplexServer(ctx context.Context, cfg *config.FulcioConfig, ctClient *
 	)
 
 	// GRPC server
-	grpcCAServer := server.NewGRPCCAServer(ctClient, baseca, ip)
+	grpcCAServer := server.NewGRPCCAServer(ctClient, baseca, algorithmRegistry, ip)
 	protobuf.RegisterCAServer(d.Server, grpcCAServer)
 	if err := d.RegisterHandler(ctx, protobuf.RegisterCAHandlerFromEndpoint); err != nil {
 		return fmt.Errorf("registering grpc ca handler: %w", err)
```

Comment:
Can we either add a comment that the legacy path (`NewLegacyGRPCCAServer) will not be updated to support this feature, or if it's not much work, can we add it? I'm OK with either approach, with a slight preference for uniformity between the two if it's not much work.

Reply:
Maybe I'm missing something, but I see that the legacy grpc server creates a v2 request from the legacy request (https://github.com/sigstore/fulcio/blob/main/pkg/server/legacy_server.go#L100) and then the new logic takes place anyway. So this is automatically supported on the legacy grpc server as well.

Are you referring to any edge case I'm not considering?